studio.assetmanager.ams.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: studio.assetmanager.ams
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: ams
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # Access to network
+  - --share=network
+  # Access to X11
+  - --socket=x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
+  # Access to all files
+  - --filesystem=host
+  # Access to OpenGL
+  - --device=dri
+  # Use same mouse cursors as host
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.secrets
+
+rename-desktop-file: asset-manager-studio.desktop
+rename-icon: asset-manager-studio
+modules:
+  - name: ams-desktop
+    buildsystem: simple
+    build-commands:
+      # Extract .deb & cleanup
+      - ar -x asset-manager-studio*.deb
+      - rm -f asset-manager-studio*.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+
+      # Copy application to Flatpack destination, and then clean it up
```

Comment:
```suggestion
      # Copy application to Flatpak destination, and then clean it up
```

Reply:
Sorry, first time using Flatpak-builder,   It is quite the learning curve especially when the messages are non-explanatory on what is broken.    I realize the comment isn't terribly useful, but a year from now having comments that even state the obvious makes reading the yaml file easier to find whatever area I need to find.